com.reprezen.swagedit/src/com/reprezen/swagedit/editor/hyperlinks/JsonReferenceHyperlinkDetector.java

```diff
@@ -37,42 +49,102 @@ protected boolean canDetect(String basePath) {
 
 	@Override
 	protected IHyperlink[] doDetect(SwaggerDocument doc, ITextViewer viewer, HyperlinkInfo info, String basePath) {
-		String label = null;
+		if (info.text.matches(LOCAL_REF_PATTERN.pattern())) {
+			return doDetectLocalLink(doc, viewer, info);
+		} else {
+			return doDetectExternalLink(info);
+		}
+	}
+
+	private IHyperlink[] doDetectExternalLink(HyperlinkInfo info) {
```

Comment:
I modified the expanded petstore example with external references like this:
<img width="954" alt="screen shot 2016-05-18 at 4 33 11 pm" src="https://cloud.githubusercontent.com/assets/644582/15374215/23f68dc8-1d17-11e6-8b68-b6a37af07186.png">

and the navigation does not work. 

I understand that this is a corner case. What can I do to make it work?